.travis.yml

```diff
@@ -1,31 +0,0 @@
-language: dart
-
-jobs:
-  include:
-    # OverReact Lib CI Jobs
-    - dart: stable
-      name: "OverReact Lib (SDK: stable)"
-      script:
-        - ./tool/travis_quality.sh
-        - ./tool/travis_tests.sh
-    - dart: dev
-      name: "OverReact Lib (SDK: dev)"
-      script:
-        - ./tool/travis_quality.sh
-        - ./tool/travis_tests.sh
-    # Analyzer Plugin CI Jobs
-    - dart: "2.7.2"
-      name: "Analyzer Plugin (SDK: 2.7)"
-      script: dart tool/travis_link_plugin_deps.dart && cd ./tools/analyzer_plugin && ./tool/travis.sh
```

Comment:
Was there a still valid reason to also run analyzer plugin CI on Dart 2.7.2?

Reply:
Running Github CI on a specific version of Dart is not something I was successful doing... If @greglittlefield-wf feels we should still run the analyzer plugin on 2.7.2, I'll have to look into another option (possibly using Workiva build or something).

Reply:
I think we should still run in 2.7.2 since that's what most of our consumers use for local dev